homeassistant/components/zha/binary_sensor.py

```diff
@@ -91,6 +91,14 @@ def is_on(self) -> bool:
             return False
         return self._state
 
+    @property
+    def name(self):
+        """Return the name of the number entity."""
```

Comment:
I don't think this should be `number`
```suggestion
        """Return the name of the binary sensor entity."""
```